src/lib/src/content-search-dialog/content-search-dialog.service.ts

```diff
@@ -21,9 +22,13 @@ export class ContentSearchDialogService {
     private contentSearchDialogConfigStrategyFactory: ContentSearchDialogConfigStrategyFactory,
     private mimeResizeService: MimeResizeService) { }
 
+  get isContentSearchDialogOpen(): BehaviorSubject<boolean> {
```

Comment:
You shouldn't expose the subject. Use a Observable instead

Reply:
You are right. Fixed.